docs/admin/authentication.md

```diff
@@ -81,6 +81,59 @@ provider external to Kubernetes.  We plan to make it easy to develop modules
 that interface between Kubernetes and a bedrock authentication provider (e.g.
 github.com, google.com, enterprise directory, kerberos, etc.)
 
+## APPENDIX
+
+### Make Certificates
+
+When using client certificate authentication, you should generate certificates by
+deployment script or some commands.
+
+**Deployment script** is implemented at
+`cluster/saltbase/salt/generate-cert/make-ca-cert.sh`.
+Execute this script with two parameters. First is the `cert_ip`, second is the
```

Comment:
"The first is the IP address of the apiserver; the second is a list of [subject alternate names](http://wiki.cacert.org/FAQ/subjectAltName) in the form `IP:<ip-address>` or `DNS:<dns-name>`"